runtime/src/lib.rs

```diff
@@ -146,7 +146,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     //   `spec_version`, and `authoring_version` are the same between Wasm and native.
     // This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
     //   the compatible custom types.
-    spec_version: 197,
+    spec_version: 198,
```

Comment:
We should bump this to whatever finney is , otherwise the build wont go all the way thorugh , as we would have to manually bump this. Not unless its a hotfix into testnet.